template/.gitignore

```diff
@@ -18,4 +18,10 @@
 .glide/
 coverage/
 
+# JUnit test output
+junit.xml
+
+# CI related files
 version.yaml
+ci/.image_tag
+ci/.project_name
```

Comment:
We should either remove the ci files from this .gitignore or remove the ci/.gitignore file.